delorean/interface.py

```diff
@@ -35,7 +59,13 @@ def parse(s, dayfirst=True, yearfirst=True):
         # delorean object that represents the time.
         do = Delorean(datetime=dt, timezone=UTC)
     return do
-
+try:
+    parse.__doc__ = parse.__doc__.format(mode_utc=NAIVEMODE_UTC,
```

Comment:
Because they are constants.  I generally don't like replicating values of constants twice.  What's the negative in the way it is currently done?  To me it is the right way, but it's your code, so whatever. :)